### DISCUSSION TITLE
Ignore MQTT discovery of specific devices valueIds

### DISCUSSION THREAD

**Discussion by Olivier6767** [zwave-js/zwave-js-ui#3365] [Q&A]
I have around 30 Z-Wave devices using an Aeotec ZW090 stick.
I have this setup sine several years, using Domoticz and currently transitioning to Home Assistant.
The issue is that for many nodes , I get several 'phantom' devices being discovered.
These will then generate useless devices in Domoticz or HomeAssistant.

![Phantom devices](https://github.com/zwave-js/zwave-js-ui/assets/28684355/98c2d1db-7168-474f-9b58-ba632d602fb6)

I have tried many times to delete these devices in Zwave-JS-UI by selecting the device, pressing DELETE than STORE.
I have also cleared the MQTT messages with RETAIN flag, and also tried clearing the full MQTT database.
However each time I restart Zwave-js-ui, these phantom devices come back.

I also noticed that each time I restart, the 'phantom' devices are always the same.
I have multiple nodes of the same type (for example Fibaro FGWPE/F-101), but they don't have the same 'phantom' devices, some have less, some have more.

Why is this happening?
Is there a place within Zwave-JS-UI or it's file structure that I need to clear or delete ?
How can I avoid this?

Thanks!

**Comment by robertsLando** [comment]
@Olivier6767 You shouldn't remove them from that list, you should just disable the discovery of the values you don't want to discover

**Comment by Olivier6767** [comment]
How do I disable the discovery for just these values?
I have the option to disable discovery for the whole node, but then I will be missing the 'good' values as well.

**Comment by robertsLando** [comment]
You should set `"ignoreDiscovery": true` in the discovery configuration of the devices you don't want to discover

**Comment by MadPatrick** [comment]
Hi.

Just a small question

If i disable the devices and press "Store" will this keep Domoticz clear form discovering these devices after a restart ?

<img width="853" height="411" alt="image" src="https://github.com/user-attachments/assets/06769a17-38c6-4cfc-976d-ee4cf899951e" />
